cmds/modules/zui/header.go

```diff
@@ -77,10 +79,33 @@ func headerRenderer(ctx context.Context, c zbus.Client, h *widgets.Paragraph, r
 				cache = red("no ssd disks detected")
 			}
 
-			h.Text = fmt.Sprintf(s, nodeID, farm, version.String(), env.RunningMode.String(), cache)
+			var utsname syscall.Utsname
+			var uname string
+			if err := syscall.Uname(&utsname); err != nil {
+				uname = red(err.Error())
+			} else {
+				uname = green(int8ToStr(utsname.Release[:]))
+			}
+
+			h.Text = fmt.Sprintf(s, nodeID, farm, version.String(), env.RunningMode.String(), uname, cache)
 			r.Signal()
 		}
 	}()
 
 	return nil
 }
+
+func int8ToStr(arr []int8) string {
+	if len(arr) == 0 {
+		return ""
+	}
+
+	b := make([]byte, 0, len(arr))
+	for _, v := range arr {
+		if v == 0 {
+			break
+		}
+		b = append(b, byte(v))
+	}
+	return string(b)
+}
```

Comment:
Since we know that int8 is a byte we can actually do this without copying the data. In C this would be a simple cast but this is not supported in Go

But Go still provide unsafe pointers type so this can be done like this instead
```suggestion
func int8ToStr(arr []int8) string {
        // get a pointer to the slice (this is like void* in C)
        ptr := unsafe.Pointer(&arr)
        // recast the pointer to a byte array pointer then get the value (deref) of it
        // so bytes is now a []byte not &[]byte
        bytes := *(*[]byte)(ptr)
        // create string
        return string(bytes)
        // this can be all done in one liner
        return string(*(*[]byte)(unsafe.Pointer(&x)))
}
```

Reply:
test it before u update the PR